Tests/StoreKitUnitTests/StoreKit2/StoreKit2TransactionListenerTests.swift

```diff
@@ -136,7 +136,7 @@ class StoreKit2TransactionListenerTests: StoreKit2TransactionListenerBaseTests {
         let (purchaseResult, _, purchasedTransaction) = try await self.purchase()
 
         let sk2Transaction = try await self.listener.handle(purchaseResult: purchaseResult)
-        expect(sk2Transaction.transaction) == purchasedTransaction
+        expect(sk2Transaction.transaction?.sk2Transaction) == purchasedTransaction
```

Comment:
I'm confused about this. Why are we doing `sk2Transaction.transaction?.sk2Transaction`?

Reply:
The first `sk2transaction` variable was poorly named. `handle(purchaseResult)` returns a tuple and we changed the type of one of its fields from `SK2Transaction` to `StoreTransaction` so that's why we needed to change this test. Renamed the variable to `resultData`